tests/apply.test.js

```diff
@@ -292,29 +292,26 @@ test('@apply classes from outside a @layer', async () => {
       }
 
       .bar {
-        font-weight: 700;
         --tw-text-opacity: 1;
         color: rgba(239, 68, 68, var(--tw-text-opacity));
+        font-weight: 700;
       }
 
       .bar:hover {
         --tw-text-opacity: 1;
-        color: rgba(16, 185, 129, var(--tw-text-opacity));
+        color: rgba(34, 197, 94, var(--tw-text-opacity));
```

Comment:
The color value looks odd, but that's because of a rebase and the colors in #5384 has been merged since. 
The "old" value is `text-emerald-500` and the new value is `text-green-500`